README.md

```diff
@@ -1,24 +1,31 @@
 FileReader
-===========
+==========
 
-Read contents of a file.
+Reads contents of a file.
+
+Input signal trigger a read from a file. The contents of that file are placed on the notified signal in a configurable attribute.
 
 Properties
---------------
-None
+----------
+
+-   **File** (expr): Location of file to read from. Path must be absolute.
+-   **File Field** (str): New attribute name to add to Signal that contains the **File**.
```

Comment:
I think this is actually the filename, rather than the actual file object, is that right? Should clarify here